frontend/src/components/object/ObjectTable.vue

```diff
@@ -79,18 +79,14 @@ const showInfo = async (id: string) => {
     </Column>
     <Column header="Actions" headerStyle="width: 200px" headerClass="header-right" bodyClass="content-right actions-buttons">
       <template #body="{ data }">
-        <Button class="p-button-lg p-button-text">
-          <font-awesome-icon icon="fa-solid fa-download" />
-        </Button>
+        <DownloadObjectButton :mode="ButtonMode.ICON" :ids="['1']" />
```

Comment:
is :ids="['1']" a placeholder?
are you going to do something like :ids="[data.id]" and then if multiple are selected push them all to a single array?